fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java

```diff
@@ -671,13 +671,6 @@ private void executeByNereids(TUniqueId queryId) throws Exception {
                                 + Env.getCurrentEnv().getSelfNode().getHost() + ") and failed to execute"
                                 + " because Master FE is not ready. You may need to check FE's status"));
                     }
-                    if (context.getSessionVariable().isEnableInsertGroupCommit()) {
```

Comment:
Here is the logic of if forward group commit, should not remove it, because your PR does make group commit not fallback